### PR TITLE
Updated installer flags/options for 6.5

### PR DESCRIPTION
**Pull request by vijay8451**
Updated installer options/flags:
 
robottelo]# pytest -v tests/foreman/installer/test_installer.py -k test_installer_options_and_flags
============================================================================ test session starts ============================================================================
platform linux -- Python 3.4.9, pytest-3.6.1, py-1.7.0, pluggy-0.6.0 -- /usr/bin/python3.4
cachedir: .pytest_cache
metadata: {'Python': '3.4.9', 'Platform': 'Linux-3.10.0-693.11.6.el7.x86_64-x86_64-with-redhat-7.3-Maipo', 'Plugins': {'ordering': '0.6', 'mock': '1.10.0', 'metadata': '1.7.0', 'services': '1.3.0', 'html': '1.19.0'}, 'Packages': {'py': '1.7.0', 'pytest': '3.6.1', 'pluggy': '0.6.0'}}
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/vijsingh/my_projects/ROBO/master/robottelo, inifile:
plugins: ordering-0.6, html-1.19.0, metadata-1.7.0, mock-1.10.0, services-1.3.0
collecting 4 items                                                                                                                                                          2018-12-11 13:33:37 - conftest - DEBUG - Collected 4 test cases

collected 4 items / 3 deselected                                                                                                                                            

tests/foreman/installer/test_installer.py::test_installer_options_and_flags PASSED                                                                                    [100%]

================================================================== 1 passed, 3 deselected in 13.53 seconds ==================================================================

### CODE REVIEW

File: tests/foreman/installer/test_installer.py [hunk 1]
```diff
@@ -902,10 +1054,19 @@
     u'--reset-foreman-server-ssl-crl',
     u'--reset-foreman-server-ssl-key',
     u'--reset-foreman-server-ssl-port',
+    u'--reset-foreman-server-ssl-protocol',
     u'--reset-foreman-serveraliases',
     u'--reset-foreman-servername',
     u'--reset-foreman-ssl',
+    u'--reset-foreman-telemetry-logger-enabled',
```
Comment (ntkathole) [inline tests/foreman/installer/test_installer.py]: @san7ket can you please take a look if all options are valid for telemetry?

File: tests/foreman/installer/test_installer.py [hunk 2]
```diff
@@ -1103,8 +1307,9 @@
     u'--reset-puppet-version',
     u'--scenario',
     u'--skip-checks-i-know-better',
+    u'--skip-puppet-version-check',
     u'--upgrade',
-    u'--upgrade-puppet',
+    u'--upgrade-mongo-storage-engine',
```
Comment (ntkathole) [inline tests/foreman/installer/test_installer.py]: @alda519 Is this option is regarding new mongo upgrade feature?

File: tests/foreman/installer/test_installer.py [hunk 3]
```diff
@@ -793,6 +925,17 @@
     u'--reset-foreman-proxy-oauth-consumer-key',
     u'--reset-foreman-proxy-oauth-consumer-secret',
     u'--reset-foreman-proxy-oauth-effective-user',
+    u'--reset-foreman-proxy-plugin-ansible-ansible-dir',
+    u'--reset-foreman-proxy-plugin-ansible-enabled',
+    u'--reset-foreman-proxy-plugin-ansible-host-key-checking',
+    u'--reset-foreman-proxy-plugin-ansible-listen-on',
+    u'--reset-foreman-proxy-plugin-ansible-working-dir',
+    u'--reset-foreman-proxy-plugin-dhcp-infoblox-dns-view',
```
[the remark this answers is not in the document]
Comment (pgagne) [reply tests/foreman/installer/test_installer.py]: @ntkathole yes for the the `-{dhcp,dns}-infoblox` ones.

File: tests/foreman/installer/test_installer.py (hunk 2 above)
Comment (alda519) [reply tests/foreman/installer/test_installer.py]: yes, it is
see https://bugzilla.redhat.com/show_bug.cgi?id=1593253

**Pull request merged**